### PR TITLE
Pbi 254312 projectile rebranding

### PR DESCRIPTION
**Pull request by VictoriaTouati-AVEVA**
This work was done for the UI rebranding. Screenshots have been updated to reflect the new UI. In several topics, screenshots were removed if not essential. Some of the steps needed minor updating/rewording due to new UI elements.

### CODE REVIEW

File: content/overview/get-started/introduction-portal-interface.md [hunk 1]
```diff
@@ -38,23 +38,21 @@ Most pages consist of two panes:
...
-
-The menu also provides a link to a page where you can give feedback about OCS.
-
-![User and tenant information](../images/user-and-tenant.png "User and tenant information")
+| Numbered Item | Function |
+|---------------|----------|
+| 1 - Change Namespace dropdown menu |Click to select a different namespace. |
+| 2 - View Notifications icon |Click to view  OCS-generated notifications. | 
+| 3 - Question mark icon | Click to access the online documentation. |
+| 4 - User profile icon  | Click to view a menu of the following commands: The currently logged in user, tenant details, resource usage, feedback page, and sign out of the portal. |
```
Comment (rmuolic) [inline content/overview/get-started/introduction-portal-interface.md:57]:
```suggestion
| 2 - View Notifications icon | Click to view OCS-generated notifications. | 
| 3 - Question mark icon | Click to access the online documentation. |
| 4 - User profile icon | Click to view a menu of the following commands: The currently logged in user, tenant details, resource usage, feedback page, and sign out of the portal. |
```

File: content/overview/get-started/gs-metadata-rules.md [hunk 2]
```diff
@@ -94,24 +88,21 @@ On the Define Mappings page, you see the following. Below the stream pattern are
...
-6. In the **Map To** text box, enter **San Leandro**. 
-
-7.  You can create additional mappings for streams which are not yet stored in OCS but which you anticipate in the future. Click **Add Mappings** to add another mapping.
+1.  You can create additional mappings for streams which are not yet stored in OCS but which you anticipate in the future. Click **Add Mappings** to add another mapping.
 
-8. In the **Map From** text box, enter **PHL**. 
+1. In the **Map From** text box, enter **PHL**. 
 
-9. In the **Map To** text box, enter **Philadelphia**.
+1. In the **Map To** text box, enter **Philadelphia**.
```
Comment (rmuolic) [inline content/overview/get-started/gs-metadata-rules.md:101]: Same here, why enter a particular city name?

File: content/overview/get-started/gs-asset-types.md [hunk 3]
```diff
@@ -53,20 +53,21 @@ Follow the procedure to use the asset explorer to create an asset type.
 1. In the asset type pane, use the icons in the upper right of the toolbar to switch between the following:
 
      -  ![Card view](images/card-view.png) &ndash; Card view displays the asset types as a series of tiles
```
Comment (rmuolic) [inline content/overview/get-started/gs-asset-types.md:55]:
```suggestion
     -  ![Card view](images/card-view.png) &ndash; Card view displays the asset types as a series of tiles.
```

File: content/overview/get-started/gs-asset-types.md [hunk 4]
```diff
@@ -53,20 +53,21 @@ Follow the procedure to use the asset explorer to create an asset type.
 1. In the asset type pane, use the icons in the upper right of the toolbar to switch between the following:
 
      -  ![Card view](images/card-view.png) &ndash; Card view displays the asset types as a series of tiles 
-     -  ![Table view](images/table-view.png) &ndash; Table view displays the asset types in a table format 
+     -  ![Table view](images/table-view.png) &ndash; Table view displays the asset types in a table format
```
Comment (rmuolic) [inline content/overview/get-started/gs-asset-types.md:56]:
```suggestion
     -  ![Table view](images/table-view.png) &ndash; Table view displays the asset types in a table format.
```

File: content/add-organize-data/store-data/types/types-procedure.md [hunk 5]
```diff
@@ -12,13 +12,13 @@ After you have created a type, it cannot be modified.
...
-1. From the **Streams** drop-down list, select **Types**. ![Streams list](../images/sds-types-procedure-types-list.png)
+1. In the left corner, click **Streams** > **Types**.
 
 1. Click **Add Type**.
 
-1. In the `Add Type` window, enter the following information:
+1. In the `Add Type` pane, enter the following information:
 
    - `Id` &ndash; Id for referencing the type
    - `Name` &ndash; (Optional) User-friendly name. If you do not provide a name, the Id will be assigned as the name.
```
Comment (rmuolic) [inline content/add-organize-data/store-data/types/types-procedure.md:24]:
```suggestion
   - `Id` &ndash; Id for referencing the type
   
   - `Name` &ndash; (Optional) User-friendly name. If you do not provide a name, the Id will be assigned as the name.
```

File: content/add-organize-data/organize-data/metadata-rules/metadata-rules-procedure.md [hunk 6]
```diff
@@ -12,15 +12,18 @@ To create a new metadata rule, follow these steps:
 
 1. In the left pane, select **Data Management** > **Sequential Data Store**.
 2. On the `Sequential Data Store` page, click **Streams**.
```
Comment (rmuolic) [inline content/add-organize-data/organize-data/metadata-rules/metadata-rules-procedure.md:14]:
```suggestion

1. On the `Sequential Data Store` page, click **Streams**.
```

File: content/add-organize-data/organize-data/metadata-rules/metadata-rules-procedure.md [hunk 7]
```diff
@@ -12,15 +12,18 @@ To create a new metadata rule, follow these steps:
...
-3. In the **Namespace** drop-down list, select the namespace in which you intend to create the metadata rule.
+3. Click the **Change Namespace** button in the upper-right toolbar, then select the namespace in which you want to create the stream.
```
Comment (rmuolic) [inline content/add-organize-data/organize-data/metadata-rules/metadata-rules-procedure.md:15]:
```suggestion

1. Click the **Change Namespace** button in the upper-right toolbar, then select the namespace in which you want to create the stream.
```

File: content/add-organize-data/organize-data/metadata-rules/metadata-rules-procedure.md [hunk 8]
```diff
@@ -12,15 +12,18 @@ To create a new metadata rule, follow these steps:
...
 4. Expand the `Streams` pane if necessary to copy the full name of the stream that will be the basis for your metadata rule.
```
Comment (rmuolic) [inline content/add-organize-data/organize-data/metadata-rules/metadata-rules-procedure.md:16]:
```suggestion

1. Expand the `Streams` pane if necessary to copy the full name of the stream that will be the basis for your metadata rule.
```

File: content/add-organize-data/organize-data/metadata-rules/metadata-rules-procedure.md [hunk 9]
```diff
@@ -12,15 +12,18 @@ To create a new metadata rule, follow these steps:
...
-
-    ![ ](../../images/expand-pane.png)
-
-5. Click the menu icon to display the menu again, and select **Metadata Management** under **Data Management**. The `Metadata Management` page displays.
-6. Click the **Namespace** drop-down list, and choose the namespace in which the model stream resides and where the metadata rule is to be created.
-7. Click **Add Metadata Rule**. The `Select Stream` page displays.
-8. Paste the copied stream name into the `Search` field. All streams in the namespace that share the name pattern of the stream you selected display.
+5. In the left pane, click **Data Management** > **Metadata Management**.
```
Comment (rmuolic) [inline content/add-organize-data/organize-data/metadata-rules/metadata-rules-procedure.md:17]:
```suggestion

1. In the left pane, click **Data Management** > **Metadata Management**.
```

File: content/add-organize-data/organize-data/metadata-rules/metadata-rules-procedure.md [hunk 10]
```diff
@@ -29,15 +32,13 @@ To create a new metadata rule, follow these steps:
 11. For each delimited section of the stream name you select, use the drop-down list to assign a metadata type: Metadata, String Literal, or Wildcard. See the table following this procedure for a description of these metadata types.
```
Comment (rmuolic) [inline content/add-organize-data/organize-data/metadata-rules/metadata-rules-procedure.md:32]:
```suggestion
1. For each delimited section of the stream name you select, use the drop-down list to assign a metadata type: Metadata, String Literal, or Wildcard. See the table following this procedure for a description of these metadata types. 
```

File: content/add-organize-data/organize-data/metadata-rules/metadata-rules-procedure.md [hunk 11]
```diff
@@ -29,15 +32,13 @@ To create a new metadata rule, follow these steps:
 11. For each delimited section of the stream name you select, use the drop-down list to assign a metadata type: Metadata, String Literal, or Wildcard. See the table following this procedure for a description of these metadata types. 
 12. After entering all metadata type entries, click **Next**.
```
Comment (rmuolic) [inline content/add-organize-data/organize-data/metadata-rules/metadata-rules-procedure.md:33]:
```suggestion
1. After entering all metadata type entries, click **Next**.
```

File: content/add-organize-data/organize-data/metadata-rules/metadata-rules-procedure.md [hunk 12]
```diff
@@ -29,15 +32,13 @@ To create a new metadata rule, follow these steps:
...
+    **Note**: If you select metadata type **Metadata**, you must also enter a metadata key. A metadata key is the key portion of a key-value pair, typically the type of data provided, such as Measurement. This is helpful when streams that match the rule provide different types of data. Entries in the **Map To** portion of mapping represent the value part of the key-value pair. For example, Power would be the defined key, while different types of data could be Demand, TOTAL/Electricity, or Metering. The mapping entries specify the labels you want to display for each of those values.
     
-    ![ ](../../images/define-mappings.png)
-
 13. In the `Define Mappings` page, choose one of the following options:
```
Comment (rmuolic) [inline content/add-organize-data/organize-data/metadata-rules/metadata-rules-procedure.md:37]:
```suggestion
1. In the `Define Mappings` page, choose one of the following options:
```

File: content/add-organize-data/organize-data/metadata-rules/metadata-rules-procedure.md [hunk 13]
```diff
@@ -29,15 +32,13 @@ To create a new metadata rule, follow these steps:
...
      - To display the raw stream data for the specified metadata key, select **Copy Values** under **Mapping Type**.
      - To assign a label to data values, select **Map Values** and click **Generate Mappings**.
 14. If you selected **Map Values**, then do the following: In the **Map To** field, enter the label you want displayed for each defined metadata key, and click **Next**.
```
Comment (rmuolic) [inline content/add-organize-data/organize-data/metadata-rules/metadata-rules-procedure.md:40]:
```suggestion

1. If you selected **Map Values**, then do the following: In the **Map To** field, enter the label you want to display for each defined metadata key, and click **Next**.
```

File: content/add-organize-data/organize-data/metadata-rules/metadata-rules-procedure.md [hunk 14]
```diff
@@ -29,15 +32,13 @@ To create a new metadata rule, follow these steps:
...
-15. (Optional) Click **Add Mapping** to define a mapping for any other stream name part.
+15. Optional: Click **Add Mapping** to define a mapping for any other stream name part.
```
Comment (rmuolic) [inline content/add-organize-data/organize-data/metadata-rules/metadata-rules-procedure.md:41]:
```suggestion

1. Optional: Click **Add Mapping** to define a mapping for any other stream name part.
```

File: content/add-organize-data/organize-data/metadata-rules/metadata-rules-procedure.md [hunk 15]
```diff
@@ -29,15 +32,13 @@ To create a new metadata rule, follow these steps:
...
 16. Click **Next**. All matching streams for the rule are displayed.
 17. Enter the metadata rule **Name** and **Description** in the `Preview and Run` page.
 18. Click **Save & Execute**.
```
Comment (rmuolic) [inline content/add-organize-data/organize-data/metadata-rules/metadata-rules-procedure.md:44]:
```suggestion

1. Click **Next**. All matching streams for the rule are displayed.

1. Enter the metadata rule **Name** and **Description** in the `Preview and Run` page.

1. Click **Save & Execute**.
```

File: content/add-organize-data/organize-data/assets/create-asset-rule.md [hunk 16]
```diff
@@ -101,7 +101,7 @@ In this step, you specify the naming pattern used to find and match the appropri
 
    The token is assigned the name `site`. 
 
-   ![First token](images/first-token.png)
+   <!-- WRITER'S NOTE: All the screen captures need to be reviewed and probably updated. In particular, the screen captures in this "Step 1: Extract the tokens from the stream" will need to be updated. The wording of the choices is being changed. -->
```
Comment (rmuolic) [inline content/add-organize-data/organize-data/assets/create-asset-rule.md:104]: Should there be an updated screen capture here?

File: content/add-organize-data/organize-data/assets/create-asset-rule.md [hunk 17]
```diff
@@ -111,12 +111,12 @@ In this step, you specify the naming pattern used to find and match the appropri
 
    **Note:** There are other matching options which could be used with the example streams, `the string literal "Tank"` or `the next 4 letters`. This rule needs to identify equipment other than tanks and equipment with names that are not 4 letters long, so these are not good choices. 
 
-   ![Second token](images/second-token.png) 
-   
+   <!--WRITER'S NOTE: These images were taken with SnagIt and reduced by 50%. The resolution may be too fuzzy and they may have to be enlarged. -->
```
Comment (rmuolic) [inline content/add-organize-data/organize-data/assets/create-asset-rule.md:114]: Should there be an updated screen capture here?

**Review by rmuolic — CHANGES_REQUESTED** [review]
Minor comments and suggested changes from my review.

File: content/add-organize-data/organize-data/assets/create-asset-rule.md (hunk 17 above)
Comment (VictoriaTouati-AVEVA) [reply content/add-organize-data/organize-data/assets/create-asset-rule.md:114]: Took this graphic out. I think it's ok as is...will check w/writer (AF?) on Monday.

File: content/add-organize-data/organize-data/assets/create-asset-rule.md (hunk 16 above)
Comment (VictoriaTouati-AVEVA) [reply content/add-organize-data/organize-data/assets/create-asset-rule.md:104]: I don't think the screen capture is necessary. Will consult w/writer (AF).

File: content/overview/get-started/gs-types.md [hunk 18]
```diff
@@ -77,17 +76,17 @@ After creating the namespace, the next step is to add one or more types. For mor
...
 
-1. Click the checkbox to select the **MyOCS.PumpState** type in the list. Click **More options** and click **Manage Permissions**.
+1. Click the checkbox to select the **MyOCS.PumpState** type in the list. Click the ![More Options button](images/more-options.png) icon, then click **Manage Permissions**.
 
     This opens the `Manage Permissions for MyOCS.PumpState` window where you can override the default permissions and set permissions for the specific type. You must be assigned the Tenant Administrator role to configure the type permissions.
 
     Review the permissions for the MyOCS.PumpState type, and when you have finished exploring this window, click **Cancel** to continue. 
 
-1. Click **More options** and click **View Type**.
+1. Click the ![More Options button](images/more-options.png) icon, then click **View Type**. <!--VTT, 9/22/21: It appears this step is no longer needed + the View Type command is not on the More Options menu. After selecting a type, it's properties are shown in the left pane.-->
```
[the remark this answers is not in the document]
Comment (VictoriaTouati-AVEVA) [reply content/overview/get-started/gs-types.md]: Deleted.

File: content/overview/get-started/gs-metadata-rules.md [hunk 19]
```diff
@@ -94,24 +88,21 @@ On the Define Mappings page, you see the following. Below the stream pattern are
 
      A stream (for example, MyOCS.WestCoast.Mfg.Inventory) can match both patterns, but the values ("WestCoast," "Mfg") are mapped for only one of the metadata rules (MyOCS.{Location}.{Department}.\*). Therefore, if you execute the MyOCS.{Region}.{Site}.\* rule, it will not generate metadata for the MyOCS.WestCoast.Mfg.Inventory stream.
 
-5. To generate mappings from existing streams, click **Generate Mappings**.
+1. To generate mappings from existing streams, click **Generate Mappings**.
 
-   OCS searches and finds any existing streams that match the pattern (in this example, the **MyOCS.{Region}.{Site}.*** pattern). For each stream that matches the pattern, it lists that part of the stream name, in this example, it lists the part that corresponds to the **Site** key.
+   OCS searches and finds any existing streams that match the pattern (the **MyOCS.{Region}.{Site}.*** pattern). For each stream that matches the pattern, it lists that part of the stream name, and in this example, it lists the part that corresponds to the **Site** key.
    
+1. In the **Map To** text box, enter **San Leandro**.
```
[the remark this answers is not in the document]
Comment (VictoriaTouati-AVEVA) [reply content/overview/get-started/gs-metadata-rules.md:95]: I have no clue. I did not author this topic.

File: content/overview/get-started/gs-metadata-rules.md (hunk 2 above)
Comment (VictoriaTouati-AVEVA) [reply content/overview/get-started/gs-metadata-rules.md:101]: I have no clue. I did not author this topic.

**Review by rmuolic — APPROVED** [review]

**Pull request merged**